transform_migrate.py

```diff
@@ -0,0 +1,366 @@
+import os
+import awswrangler as wr
+import pandas as pd
+import numpy as np
+from glue import sales_val_flagging as flg
+import subprocess as sp
+
+# Set working dir to manual_update, standardize yaml and src locations
+root = sp.getoutput("git rev-parse --show-toplevel")
+os.chdir(os.path.join(root))
+
+
+def write_dfs_to_s3(dfs, bucket, table):
+    """
+    Writes dicctionary of dfs to bucket
+    """
+
+    for df_name, df in dfs.items():
+        file_path = f"{bucket}/sale/{table}/{df_name}.parquet"
+
+        wr.s3.to_parquet(df=df, path=file_path, index=False)
+
+
+def read_parquets_to_dfs(prefix, table):
+    """
+    Reads all parquet files from a specified S3 path into separate pandas DataFrames,
+    names them with a given prefix, and returns them in a dictionary.
+
+    Parameters:
+    - prefix: The prefix to prepend to each DataFrame name.
+    - s3_path: The S3 path where the parquet files are stored. Should be in the format "s3://bucket-name/path/to/files".
+
+    Returns:
+    A dictionary of DataFrames keyed by their names.
+    """
+    # List all parquet files in the specified S3 path
+    parquet_files = wr.s3.list_objects(
+        os.path.join(os.getenv("AWS_S3_WAREHOUSE_BUCKET"), "sale", table),
+        suffix=".parquet",
+    )
+
+    # Initialize a dictionary to hold the dataframes
+    dfs = {}
```

Comment:
This script handles each of the transformations by `table` first (`flag/parameter/group_mean/metadata`). We transform the existing prod data, and we transform the locally saved metadata from our recent res and condos update.